torchbenchmark/models/alexnet/__init__.py

```diff
@@ -3,36 +3,32 @@
 import torch
 import torch.optim as optim
 import torchvision.models as models
+
 from ...util.model import BenchmarkModel
 from torchbenchmark.tasks import COMPUTER_VISION
 
-from torchbenchmark.util.env_check import parse_extraargs
+from torchbenchmark.util.framework.vision.args import parse_args, apply_args
 
 class Model(BenchmarkModel):
     task = COMPUTER_VISION.CLASSIFICATION
-    optimized_for_inference = True
```

Comment:
Why are we removing optimized_for_inference here?

Reply:
Currently it is not used and I thought this is redundant in metadata.yml?

Reply:
Sorry it seems `gen_summary_metadata.py` depends on this - I will revert this change. Sorry about that.